ante/tests/setup.go

```diff
@@ -1,9 +1,8 @@
-package ante_test
+package ante_tests
```

Comment:
Not really a problem, this is a black box testing convention that Ginkgo bootstraps by default.
Can you keep the package naming as is, `ante_test`?

Reply:
Also, not necessary to nest in a separate directory.
Can we keep the same, but export the package as `ante_test`?